tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/Resources.cs

```diff
@@ -97,7 +97,9 @@ internal static class Resources
 
         public static string CouldNotFindIisApplication(string site, string application) => $"Could not find IIS application \"{site}{application}\". ";
 
-        public static string IisManagerInitializationError(string error) => $"Could not initialize IIS manager: {error}. Try to run the tool in administrator mode.";
+        public static string IisManagerInitializationError(string error) => $"Could not initialize IIS manager: {error} Try to run the tool in administrator mode.";
+
+        public static string IisWorkerProcessError(string error) => $"Could not detect the worker process: {error} Try to run the tool in administrator mode.";
```

Comment:
Not sure about the suggested wording, but I feel like people will try to look for the tool's "administrator mode" otherwise 🤣 

```suggestion
        public static string IisWorkerProcessError(string error) => $"Could not detect the worker process: {error} You must run the tool from a shell running as an Administrator.";
```